lib/resources/kernel_module.rb

```diff
@@ -36,19 +68,51 @@ def loaded?
       !found.nil?
     end
 
-    def version
-      if inspec.os.redhat? || inspec.os.name == 'fedora'
-        modinfo_cmd = "/sbin/modinfo -F version #{@module}"
-      else
-        modinfo_cmd = "modinfo -F version #{@module}"
-      end
+    def disabled?
+      !modprobe_output.match(%r{^install\s+#{@module}\s+/(s?)bin/(true|false)}).nil?
+    end
 
-      cmd = inspec.command(modinfo_cmd)
+    def disabled_via_bin_true?
+      !modprobe_output.match(%r{^install\s+#{@module}\s+/(s?)bin/true}).nil?
+    end
+
+    def disabled_via_bin_false?
+      !modprobe_output.match(%r{^install\s+#{@module}\s+/(s?)bin/false}).nil?
+    end
+
+    def blacklisted?
```

Comment:
Is checking for blacklist enough? A module can be disabled via `/bin/true` too. We could make `disabled_via_bin_false` and `disabled_via_bin_true?` private and add the check to `blacklisted?`.